### PR TITLE
feat: add transfer-security-tokens-partition processor

### LINKED ISSUE #13: Make TransferSecurityTokensPartition Processor and command
## Expected Behavior
- Make TransferSecurityTokensPartition Processor
- Make TransferSecurityTokensPartition Command
## Actual Behavior
- not implemented
## Specifications
  - commit; fc4ff1ece3cc7dabc83ade54066134e7131a9c80
  - os; macOS Monterey 12.6
  - go version;  go1.19 darwin/arm64

### PR DESCRIPTION
**Pull request by wyuinche**
### Pull-request Type (fix, feat, bug, doc, chore, test, etc...) 

- feature

### Current Behavior (Link to an open issue)

- Close #13 

### New Behavior (if this is a feature change)

- add transfer security tokens partition processor

### CODE REVIEW

File: sto/transfer_security_tokens_partition_process.go [hunk 1]
```diff
@@ -0,0 +1,460 @@
...
+		ip := transferSecurityTokensPartitionItemProcessorPool.Get()
+		ipc, ok := ip.(*TransferSecurityTokensPartitionItemProcessor)
+		if !ok {
+			return nil, nil, e(nil, "expected TransferSecurityTokensPartitionItemProcessor, not %T", ip)
+		}
+
+		ipc.h = op.Hash()
+		ipc.sender = fact.Sender()
+		ipc.item = it
+		ipc.partitions = &partitions
```
Comment (soonkuk) [inline sto/transfer_security_tokens_partition_process.go:371]: @wyuinche I think you don't need to pass a map using '&'. A map is passed by reference, so it is unnecessary to pass it as a pointer.

Comment (wyuinche) [reply sto/transfer_security_tokens_partition_process.go]: @soonkuk It has been fixed.

File: sto/transfer_security_tokens_partition_process.go [hunk 2]
```diff
@@ -0,0 +1,460 @@
...
+		return err
+	}
+
+	if err := checkExistsState(currency.StateKeyAccount(it.Receiver()), getStateFunc); err != nil {
+		return err
+	}
+
+	if err := checkNotExistsState(extensioncurrency.StateKeyContractAccount(it.Receiver()), getStateFunc); err != nil {
+		return err
+	}
```
Comment (soonkuk) [inline sto/transfer_security_tokens_partition_process.go:67]: @wyuinche The ItemProcessor need to check if the the amount satisfies the granularity requirement. An amount is divisible by granularity by zero remainder.

**Review by soonkuk — CHANGES_REQUESTED** [review]

File: sto/transfer_security_tokens_partition_process.go (hunk 2 above)
Comment (wyuinche) [reply sto/transfer_security_tokens_partition_process.go:64]: @soonkuk The processors of transfer-sto and issue-sto operation have been modified to solve it.

Comment (soonkuk) [reply sto/transfer_security_tokens_partition_process.go:64]: @wyuinche Ok, I got it.

**Review by soonkuk — APPROVED** [review]

**Pull request merged**